### DISCUSSION TITLE
How do you change the favicon?

### DISCUSSION THREAD

**Discussion by joshua-revolutions** [necolas/react-native-web#2567] [Q&A]
How do you change the favicon?

**Comment by Dovakeidy** [comment]
Like a regular website, in you main .html file.

**Reply by joshua-revolutions** [reply]
I'm using expo (bare workflow) and when I run
```
yarn web
```

It builds into a web-build folder which also contains the index.html. 
I'm not sure how I'm supposed replace that file with a custom one. I assume it gets recreated each build. But I think I just need to create a web folder and put a custom index.html in it.
